cmake/projects/Alut/hunter.cmake

```diff
@@ -0,0 +1,27 @@
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Alut
+    VERSION
+    1.1.0-469287b-p0
+    URL
+    "https://github.com/drodin/freealut/archive/hunter-1.1.0-469287b.tar.gz"
```

Comment:
created tag: https://github.com/cpp-pm/freealut/releases/tag/1.1.0-469287b-p0 please update URL to point to the tag

Reply:
done